CHANGELOG.md

```diff
@@ -357,3 +357,4 @@
 [@Flink]: https://github.com/Flink
 [@jackcasey]: https://github.com/jackcasey
 [@sl4vr]: https://github.com/sl4vr
+[@96RadhikaJadhav]: https://github.com/96RadhikaJadhav/
```

Comment:
Don't forget to add a 1-line summary of your change on the top of the file :)

Reply:
@nunosilva800  Done.

Reply:
I toke the liberty to bring in changes from upstream and fix the conflicts.
You can pull to get the latest changes on your machine.

https://github.com/whitesmith/rubycritic/pull/384/commits/a75366fb866ca2324a717d247056432c22239f25